src/test/java/seedu/address/logic/parser/CreatePastAppointmentCommandParserTest.java

```diff
@@ -0,0 +1,80 @@
+package seedu.address.logic.parser;
+
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.time.format.DateTimeParseException;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.parser.exceptions.ParseException;
+
+class CreatePastAppointmentCommandParserTest {
+
+    @Test
+    public void parse_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new CreatePastAppointmentCommandParser().parse(null));
+    }
+
+    @Test
+    public void parse_emptyString_throwsParseException() {
+        assertThrows(ParseException.class, () -> new CreatePastAppointmentCommandParser().parse(""));
+    }
+
+    @Test
+    public void parse_invalidIndex_throwsParseException() {
```

Comment:
If this should be a Parse Exception due to invalid Index provided in user input, I believe the test input should be:
"a on/12-01-2020 diag/fever"

Reply:
Done!